database.md

```diff
@@ -27,6 +27,8 @@ Almost every modern web application interacts with a database. Laravel makes int
 
 </div>
 
+Additionally, MongoDB is supported via the mongodb/laravel-mongodb package, which is officially maintained by the PHP team at MongoDB. See the [Laravel MongoDB](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/) documentation for more information.
```

Comment:
```suggestion
Additionally, MongoDB database is supported via the mongodb/laravel-mongodb package, which is officially maintained by MongoDB. See the [Laravel MongoDB](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/) documentation for more information.
```

Reply:
I like "officially maintained by MongoDB".

Should `mongodb/laravel-mongodb` get monospace formatting, as was done in other pages?

"MongoDB database is supported" sounds a bit odd to me (especially when read aloud), since the product name already includes "DB". If we need to use "database", would "MongoDB databases are" read better?

Reply:
I am fine with only MongoDB.